tee-worker/litentry/core/data-providers/src/achainable.rs

```diff
@@ -935,67 +1016,67 @@ impl AchainableTagDeFi for AchainableClient {
 	fn uniswap_v2_user(&mut self, address: &str) -> Result<bool, Error> {
 		// Uniswap V2 trader
 		let name = "Uniswap V2 trader";
-		let chain = "ethereum";
+		let chain: Web3Network = Web3Network::Ethereum;
 		let r1 =
-			request_basic_type_with_token(self, address, name, chain, None).unwrap_or_default();
+			request_basic_type_with_token(self, address, name, &chain, None).unwrap_or_default();
 
 		// Uniswap V2 liquidity provider
 		let name = "Uniswap V2 liquidity provider";
-		let r2 = request_basic_type_with_token(self, address, name, chain, None)?;
+		let r2 = request_basic_type_with_token(self, address, name, &chain, None)?;
```

Comment:
I think this `Ok(r1 || r2)` has the same problem as above - we could send out more requests than we actually need.